test/lifecycle.js

```diff
@@ -197,6 +206,38 @@ function after () {
   return fn
 }
 
+async function createPlatform ({ t, minimumFixtures = false }) {
```

Comment:
Before refactoring, how were new platforms created in tests (just to make sure you can’t use existing logic).

By the way, did you make a typo in your commit message?

> tests are **not** total**ly** isolated

Reply:
Before refactoring, there is only the test suite `store.spec.js` that created new platforms in tests.
https://github.com/stelace/stelace/blob/22b27b56bd68e337043e9d5cfbba25097e2966b5/test/integration/api/store.spec.js#L252-L255
https://github.com/stelace/stelace/blob/22b27b56bd68e337043e9d5cfbba25097e2966b5/test/integration/api/store.spec.js#L363-L372

However the platform is only created with this request. The configuration part is missing (databases credentials, databases start, seeding). That wasn't a problem in `store.spec.js` file because the test is to check the platform is correctly initialized.

The objective of the function `createPlatform` I introduced in `lifecycle.js` is to return a fully functional platform.
This logic was present but was split between `before()` and `beforeEach()` lifecycle functions, so I refactored it into `createPlatform` function.

> By the way, did you make a typo in your commit message?
> tests are not totally isolated
Yes thanks, I made a typo.

Reply:
> However the platform was only created with those requests.

I was refering to lifecycle. You obviously need to create new platforms (with some config) to run tests. This was the shareable logic I’m talking about.

> What do you think?

OK. It seems there is a typo: `needed to reused`

Reply:
Thanks. I'll merge the PR when the test passes (commit message updated).